src/App.js

```diff
@@ -1,18 +1,66 @@
 import React from 'react';
 import './App.scss';
 
-class App extends React.Component {
-  addOne = () => {};
+export class App extends React.Component {
+  state = {
+    count: 0,
+  }
+
+  addOne = () => {
+    this.setState(prevState => ({
+      count: prevState.count + 1,
+    }));
+  };
+
+  add100 = () => {
+    this.setState(prevState => ({
+      count: prevState.count + 100,
+    }));
+  };
 
-  add100 = () => {};
+  increase = () => {
+    const { count } = this.state;
 
-  increase = () => {};
+    if (count % 5 === 0) {
+      this.add100();
+    }
+
+    this.addOne();
+  };
 
   render() {
+    const { count } = this.state;
+
     return (
-      <h1>Count: 0</h1>
+      <div className="container">
+        <h1 className="text-center m-5">
+          Count:
+          {count}
+        </h1>
+        <div className="btn-group d-flex justify-content-center">
+          <button
+            type="button"
+            className="btn rounded btn-danger m-2"
+            onClick={() => this.addOne()}
```

Comment:
якщо функція не приймає параметрів то не потрібно створювати стрілочну функцію для функції addOne
```suggestion
            onClick={this.addOne}
```